src/main/java/net/snowflake/client/jdbc/SnowflakeFileTransferAgent.java

```diff
@@ -1268,6 +1287,83 @@ private static JsonNode parseCommandInGS(SFStatement statement, String command)
     return result;
   }
 
+  /**
+   * This is API function to parse the File Transfer Metadatas from a supplied PUT call response.
+   *
+   * <p>NOTE: It only supports PUT on S3/AZURE/GCS
```

Comment:
I think if I was reading this outside of Snowflake my thought would be "what other options are there" - might be worth clarifying that `LOCAL_FS` is the one not supported.